src/ucs/profile/profile_on.h

```diff
@@ -255,32 +328,36 @@ BEGIN_C_DECLS
  * @param _req      Request pointer.
  */
 #define UCS_PROFILE_REQUEST_FREE(_req) \
-    UCS_PROFILE(UCS_PROFILE_TYPE_REQUEST_FREE, "", 0, (uintptr_t)(_req));
+    UCS_PROFILE_CTX_RECORD(ucs_profile_default_ctx_h, UCS_PROFILE_TYPE_REQUEST_FREE, \
+                           "", 0, (uintptr_t)(_req));
 
 
 /*
  * Store a new record with the given data.
  * SHOULD NOT be used directly - use UCS_PROFILE macros instead.
- *
- * @param [in]     type        Location type.
- * @param [in]     name        Location name.
- * @param [in]     param32     custom 32-bit parameter.
- * @param [in]     param64     custom 64-bit parameter.
- * @param [in]     file        Source file name.
- * @param [in]     line        Source line number.
- * @param [in]     function    Calling function name.
- * @param [in,out] loc_id_p    Variable used to maintain the location ID.
+ * @param [in]     ctx        Global profile context.
+ * @param [in]     type       Location type.
+ * @param [in]     name       Location name.
+ * @param [in]     param32    custom 32-bit parameter.
+ * @param [in]     param64    custom 64-bit parameter.
+ * @param [in]     file       Source file name.
+ * @param [in]     line       Source line number.
+ * @param [in]     function   Calling function name.
+ * @param [in,out] loc_id_p   Variable used to maintain the location ID.
```

Comment:
unneeded format change. was the intent to leave just one space between the longest name and description?